src/http/server.rs

```diff
@@ -67,8 +87,11 @@ pub async fn run_server(config_path: &PathBuf) -> std::io::Result<()> {
 
     println!("Starting server {}:{}...", ayb_conf.host, ayb_conf.port);
     HttpServer::new(move || {
+        let cors = build_cors(ayb_conf.cors.clone());
```

Comment:
Worth moving to line 87 so all the config stuff is in one block?

Reply:
The `CORS` struct from `actix-cors` doesn't implement `Send`, and the `HttpServer::new` function required the lambda to be `Send`. So that doesn't compile. (Unlike the last time, I tried it before posting the comment, not after :p)